WMIAdventure/frontend/src/js/components/battle/organisms/BattleView/BattleView.js

```diff
@@ -32,13 +32,13 @@ class BattleView extends React.Component {
                            setTranslateY={this.props.setTranslateY}>
                         <MainContainer>
                             <FlexGapContainer setMargin={'10px 0 0 0'}>
-                                <UserStateContainer />
+                                <UserStateContainer enemy />

```

Comment:
Tutaj wtedy po prostu dodawalibyśmy dwa różne komponenty. Wydaje mi się że to będzie bardziej czytelne